src/GraphQL/DoNotMapClrTypeAttribute.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace GraphQL
+{
+    /// <summary>
+    /// Indicates that <see cref="SchemaExtensions.RegisterTypeMappings(Types.ISchema)"/> should
+    /// skip this class when scanning an assembly for CLR type mappings.
```

Comment:
> this class

_This_ actually may be GraphType or CLR type. At least I do not see the reasons to not do that. Would you add such a feature?

Reply:
Hmm. Interesting idea. In general the code only scans for IGraphType. But since it does need to match it up to clr types I think that makes sense.

Reply:
certainly

Reply:
Also don't forget to update documentation.

Reply:
#2795